substrate/frame/support/Cargo.toml

```diff
@@ -30,7 +30,7 @@ sp-weights = { path = "../../primitives/weights", default-features = false}
 sp-debug-derive = { path = "../../primitives/debug-derive", default-features = false}
 sp-metadata-ir = { path = "../../primitives/metadata-ir", default-features = false}
 tt-call = "1.0.8"
-macro_magic = "0.4.2"
+macro_magic = { git = "https://github.com/gupnik/macro_magic", branch = "verbatim_export" }
```

Comment:
Will update the version once the PR is merged

Reply:
You can update it now ;)